packages/shared/src/components/sources/SourceActions/index.tsx

```diff
@@ -1,44 +1,73 @@
 import React, { ReactElement } from 'react';
-import { SourceSubscribeButton } from './SourceSubscribeButton';
```

Comment:
Looks good but maybe we should remove even the `SourceSubscribeButton` component, now it should have no use in the codebase